admin/test/test_images.py

```diff
@@ -195,7 +195,7 @@ def test_configuration(self):
         build_region = builder['region']
         build_source_ami = builder['source_ami']
         publish_regions = builder['ami_regions']
-        [provisioner] = packer_configuration["provisioners"]
+        [_] = packer_configuration["provisioners"]
```

Comment:
:question: I suggest `[_provisioner]` or just remove the entire line if the unpacked  variable is unused.